src/main/java/com/google/sps/servlets/CommentServlet.java

```diff
@@ -26,8 +39,65 @@
 /** Handles comment requests. */
 @WebServlet("/api/comment")
 public class CommentServlet extends HttpServlet {
+  private Gson gson = new Gson();
+  private final String DB_COMMENTS = "comments-collection";
+
   @Override
-  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {}
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String recipeID = request.getParameter("recipeID");
+    String json;
+
+    if (recipeID == null) {
+      return;
```

Comment:
I meant something like response.setStatus(HttpServletResponse.SC_NOT_FOUND). It will really help client side debugging when you're not getting comments to show up.